deployments/tyk/volumes/tyk-dashboard/tyk_analytics.conf

```diff
@@ -22,13 +22,16 @@
   "hash_keys": true,
   "enable_multi_org_users": true,
   "email_backend": {
-    "enable_email_notifications": false,
-    "code": "sendgrid",
+    "enable_email_notifications": true,
+    "code": "smtp",
     "settings": {
-      "ClientKey": ""
+        "SMTPAddress": "mailserver:2500",
+        "SMTPPassword": "examplepassword",
+        "SMTPUsername": "email@example.com"
     },
-    "default_from_email": "you@somewhere.com",
-    "default_from_name": "Some Person"
+    "default_from_email": "jeff@wheresmyrug.com",
+    "default_from_name": "Jeffrey (The Dude) Lebowski",
+    "dashboard_hostname": "www.tyk-test.com"
```

Comment:
Should this hostname be the one used in Tyk Demo?

Reply:
Resolved